merge_request_approvals.go

```diff
@@ -80,6 +108,58 @@ type ApproveMergeRequestOptions struct {
 	SHA *string `url:"sha,omitempty" json:"sha,omitempty"`
 }
 
+// UpdateMergeRequestProjectApprovals updates the approval configurations for a given project on GitLab.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/merge_request_approvals.html#change-configuration
+func (s *MergeRequestApprovalsService) UpdateMergeRequestProjectApprovals(pid interface{}, opt *UpdateMergeRequestProjectApprovalsOptions, options ...OptionFunc) (*MergeRequestProjectApprovals, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/approvals", url.QueryEscape(project))
+
+	req, err := s.client.NewRequest("POST", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	mrp := new(MergeRequestProjectApprovals)
+
```

Comment:
Renaming this to just `m` and removing the blank line would make the code more inline with the rest of the package.